### ISSUE TITLE
Minecolony broken by last mod pack update

### ISSUE THREAD

**Issue by Consumerx54** [Boomflex/JustNotEnoughMods2#36]
I run this mod pack on my server and everyone loves it.  They have enjoyed the minecolony mod a lot but your last update broke it to a degree.  You have taken away regular chests and replaced them with the quark chests which looks cool and all but in minecolony many builds require chests and they will ONLY accept standard minecraft chests not quark chests.  I have had to give all my users a stack of standard minecraft chests so their builder on minecolonies can finish their builds.  
   I hope a new mod pack update comes out soon since minecolonies version is like 60 versions higher than the one in the mod pack  Love the mod pack keep up the good work

**Comment by Perlo0ung** [comment]
U can obtain those chest from multiple sources, like pirateships villages etc.

**Comment by Consumerx54** [comment]
Spending real time hours looking for villages is not my idea of fun just to find the most basic item in the game.  So I will continue to give my users a stack of them as they need them.  I have also upgraded minecolonies myself to a version that at least works unlike the version in the mod pack which is very buggy.  It is a ALPHA so I expect bugs.  I have also dumped the mod server utilities since the version in this pack writes since it has the need to save server data every single minute of the day.  I don't need the extra wear and tear on my hard drive for a useless operation.  Nobody on the server over night for about 8 to 10 hours but every minute it saves server data which is very stupid.  Other than that my users and I love the mod pack
